__tests__/APP.test.js

```diff
@@ -61,3 +61,59 @@ describe("task3- GET api/topics", () => {
       });
   });
 });
+
+describe.only("task5- GET api/articles/:articleiD", () => {
+  test("returns a single object (with correct properties article) when id exists", () => {
+    return request(app)
+      .get("/api/articles/3")
+      .expect(200)
+      .then(({ body }) => {
+        // console.log("in test", body.article);
+        const article = body.article;
+        expect(article).toMatchObject({
```

Comment:
This is a great test however you're missing a check for the votes property (this is key as we use likes further down the line in the backend and use the like/votes functionality on the front end)